src/components/spatial/Spatial.js

```diff
@@ -322,10 +327,15 @@ class Spatial extends AbstractSpatialOrScatterplot {
       selections = nextLoaderSelection;
       this.layerLoaderSelections[layerDef.index] = nextLoaderSelection;
     }
+    const useTransparentColor = (!layerDef.visible
+      && typeof layerDef.visible === 'boolean')
+      || Boolean(layerDef.transparentColor);
+    const transparentColor = useTransparentColor ? [0, 0, 0] : null;
```

Comment:
Needed to add this.  Forgot about this API change as well.  I will add it to the CHANGELOG.md @keller-mark

Reply:
Fixed #1077

Reply:
Thanks!